diff --git a/bundle.Dockerfile b/bundle.Dockerfile
index 0ffbe18..a485f0e 100644
--- a/bundle.Dockerfile
+++ b/bundle.Dockerfile
@@ -32,10 +32,10 @@ LABEL io.k8s.description="Red Hat OpenShift Lightspeed - AI assistant for managi
 LABEL io.k8s.display-name="Openshift Lightspeed"
 LABEL io.openshift.tags="openshift,lightspeed,ai,assistant"
 LABEL name=openshift-lightspeed
-LABEL release=<current-version>
+LABEL release=<new-version>
 LABEL url="https://github.com/openshift/lightspeed-operator"
 LABEL vendor="Red Hat, Inc."
-LABEL version=<current-version>
+LABEL version=<new-version>
 LABEL summary="Red Hat OpenShift Lightspeed"
 
 # OCP compatibility labels
diff --git a/bundle/manifests/lightspeed-operator.clusterserviceversion.yaml b/bundle/manifests/lightspeed-operator.clusterserviceversion.yaml
index c500578..f688d06 100644
--- a/bundle/manifests/lightspeed-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/lightspeed-operator.clusterserviceversion.yaml
@@ -55,7 +55,7 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.33.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v4
     repository: https://github.com/openshift/lightspeed-operator
-  name: lightspeed-operator.v<current-version>
+  name: lightspeed-operator.v<new-version>
   namespace: openshift-lightspeed
 spec:
   apiservicedefinitions: {}
@@ -536,7 +536,7 @@ spec:
   provider:
     name: Red Hat, Inc
     url: https://github.com/openshift/lightspeed-service
-  version: <current-version>
+  version: <new-version>
   relatedImages:
     - name: lightspeed-service-api
       image: quay.io/openshift-lightspeed/lightspeed-service-api:latest
diff --git a/hack/update_bundle_catalog.sh b/hack/update_bundle_catalog.sh
index 8761708..9d3c10a 100755
--- a/hack/update_bundle_catalog.sh
+++ b/hack/update_bundle_catalog.sh
@@ -36,7 +36,7 @@ if [ -z "${YQ}" ]; then
 fi
 
 # Set the bundle version
-: "${BUNDLE_TAG:=<current-version>}"
+: "${BUNDLE_TAG:=<new-version>}"
 
 : "${OPERATOR_IMAGE:=registry.redhat.io/openshift-lightspeed-tech-preview/lightspeed-rhel9-operator@sha256:049a1a398ed87e4f35c99b36304055c7f75d0188a4d8c1726df59b5f400561e5}"
 : "${BUNDLE_IMAGE:=registry.redhat.io/openshift-lightspeed-tech-preview/lightspeed-operator-bundle@sha256:c8ba8d8b4774fdaa6037fdba8cfeff0a7ee962ebe384eabe45995c8949f76eed}"

